tests/test_sockets.py

```diff
@@ -133,6 +135,31 @@ def check_asyncio_bug(anyio_backend_name: str, family: AnyIPAddressFamily) -> No
             pytest.skip("Does not work due to a known bug (39148)")
 
 
+if sys.version_info >= (3, 14):
+
+    async def no_other_refs() -> list[object]:
```

Comment:
We don't really need to duplicate this from `test_taskgroups.py`, do we? Why not just move it to a utility module in the test suite, or something?

Reply:
I could put it in conftest.py ? happy to have a utils.py

Reply:
I've usually gone with a `utils.py` in the test suite. So long as it's structured as a package (which it is in anyio's case), it works fine.

Reply:
I chucked it into conftest.py for now - we're already importing asyncio_params from there